<launch>
    <node pkg="kato_program" type="camera_lidar_fusion" name="camera_lidar_fusion" output="screen"/>
    <rosparam command="load" file="$(find kato_program)/config/params.yaml" />
</launch>
